docker_compose/app.yaml

```diff
@@ -0,0 +1,16 @@
+services:
```

Comment:
combine all compose files into one and no need for a separate folder, compose should be on the same level as dockerfile